src/core/grid_based_algorithms/lb.cpp

```diff
@@ -948,17 +950,9 @@ inline void lb_collide_stream() {
   }
 #endif // LB_BOUNDARIES
 
-#ifdef VIRTUAL_SITES_INERTIALESS_TRACERS
-  // Safeguard the node forces so that we can later use them for the IBM
-  // particle update
-  // In the following loop the lbfields[XX].force are reset to zero
-  // Safeguard the node forces so that we can later use them for the IBM
-  // particle update In the following loop the lbfields[XX].force are reset to
-  // zero
   for (int i = 0; i < lblattice.halo_grid_volume; ++i) {
```

Comment:
I think the extra sweep is not needed, this can be included into the update loop. You should probably time this, but should be faster.